src/Stratis.Bitcoin.Cli/Program.cs

```diff
@@ -57,7 +57,7 @@ public static void Main(string[] args)
                 var commandArgList = new List<string>(argList);
 
                 // Display help if required.
-                if (string.IsNullOrWhiteSpace(command) || argList.Contains("-help") || argList.Contains("--help"))
+                if (argList.Contains("-help") || argList.Contains("--help") || string.IsNullOrWhiteSpace(command))
```

Comment:
ah sorry, what I meant was that this if{} should be the first thing in the method.